lib/models/application.coffee

```diff
@@ -694,6 +694,128 @@ getApplicationModel = (deps, opts) ->
 			throw err
 		.asCallback(callback)
 
+	###*
+	# @summary Check if rolling release is enabled for specific application
+	# @name isRollingReleaseEnabled
+	# @public
+	# @function
+	# @memberof resin.models.application
+	#
+	# @param {String|Number} nameOrId - application name (string) or id (number)
+	# @returns {Promise}
+	#
+	# @example
+	# resin.models.application.isRollingReleaseEnabled('MyApp').then(function(isEnabled) {
+	# 	console.log(isEnabled);
+	# });
+	###
+
+	exports.isRollingReleaseEnabled = (nameOrId, callback) ->
+		exports.get(nameOrId, $select: 'should_track_latest_release')
+		.get('should_track_latest_release').asCallback(callback)
+
+	###*
+	# @summary Set application to particular commit
+	# @name setApplicationToRelease
+	# @public
+	# @function
+	# @memberof resin.models.application
+	#
+	# @param {String|Number} nameOrId - application name (string) or id (number)
```

Comment:
This is missing the `@param` for the commit hash.

Reply:
I guess that this comment is no longer valid, since the commit hash isn't part of the `isTrackingLatestRelease()` method call signature